docs/en/optimizing-query-objects.md

```diff
@@ -56,12 +56,14 @@ class ArticlesQuery extends QueryObject
 
   public function postFetch(Queryable $repository, \Iterator $iterator)
   {
-    $ids = array_keys(iterator_to_array($iterator, TRUE));
```

Comment:
It's here

![snimek obrazovky porizeny 2015-10-05 15 18 34](https://cloud.githubusercontent.com/assets/158625/10281375/5b642950-6b74-11e5-8fdc-84aac54ec664.png)